hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -361,14 +361,14 @@
 
             Stimulus:
               - Configure DUT/Agent to Target/Host mode respectively
-              - Fill up the acq_fifo with data to be sent out
-              - Reset the fifo randomly after a random number of bytes shows up on acq_fifo
+              - Run read write mixed traffic.
+              - Reset each fifo randomly after a random number of bytes shows up on acq_fifo
```

Comment:
do you want to clarify in the description here that it's only between stop / start also like your description?

Reply:
Updated. I will update tx fifo reset test description in separate PR.